modules/apps/script-config.js

```diff
@@ -46,7 +46,26 @@ export default class ScriptConfig extends FormApplication
         {
             this.editor = ace.edit(html.find(".ace-editor")[0]);
             this.editor.setValue(this._getScript() || "");
-            this.editor.setOptions(mergeObject(ace.userSettings, {mode : "ace/mode/js", printMargin : 0, theme : "ace/theme/solarized_dark", readOnly : !game.user.isGM}));
+            this.editor.setOptions(mergeObject(ace.userSettings, {
+                theme : "ace/theme/solarized_dark",
+                keyboardHandler : "ace/mode/vscode",
+                printMargin : 0,
+                maxLines: 999999,
+                indentedSoftWrap: false,
+                esVersion: 13,
+            }));
+            this.editor.session.on('changeMode', function(e, session){
+                if ("ace/mode/javascript" === session.getMode().$id) {
+                    if (!!session.$worker) {
+                        session.$worker.send("setOptions", [{
+                            "esversion": 13
+                        }]);
+                    }
+                }
```

Comment:
What is this doing exactly?

Reply:
This should be good to merge but I'm still curious about this.

Reply:
this should set esversion to minimize the number of syntax errors for newer javascript, but i'm afraid that it will not work, as the ace library module is using v1.10 while the newest one is v1.32 anyway it seems to look good: 
![image](https://github.com/moo-man/WFRP4e-FoundryVTT/assets/7930626/9ffa933f-147f-4a28-9391-531bfdc2a55e)

Reply:
> but i'm afraid that it will not work
Is this needed then? Or am I misunderstanding.

Reply:
it will do no harm and in future can be beneficial